qemu/tests/cfg/hotplug_mem_negative.cfg

```
@@ -25,7 +25,7 @@
                 # On arm64, machine type can be used to request the
                 # physical address size for the VM. For backward compatibility,
                 # the default IPA size is 40bits.
-                ipa_limit_check = "cat /var/log/messages | grep "IPA Size Limit:" | awk '{print $11}'"
+                ipa_limit_check = "cat /var/log/messages | grep "IPA Size Limit:" | awk '{print $11}' | head -n 1"
```

Using double quotes to wrap double quotes will make the line not readable, also it can be simple to:
```suggestion
                ipa_limit_check = "grep 'IPA Size Limit:' /var/log/messages | head -n 1 | awk '{print $(NF-1)}'"
```

---

Good suggestion, updated.